orbit/pkg/luks/luks_linux.go

```diff
@@ -35,13 +37,34 @@ const (
 
 var ErrKeySlotFull = regexp.MustCompile(`Key slot \d+ is full`)
 
+func isInstalled(toolName string) bool {
+	path, err := exec.LookPath(toolName)
```

Comment:
Let's double check orbit's `PATH` doesn't contain folders that are world writable.

Reply:
is this to ensure we're not using a malicious version of the tool?

Reply:
Yep. Somebody with non-root privileges placing a executable that would be executed by `orbit`.

Just checking I know this is already the case so we don't need to block this PR on it.